Application/src/visuals/Message.java

```diff
@@ -167,4 +167,16 @@ public int getToNode(){
         return toNode;
     }
 
+	@Override
+	public String getName() {
+		// Not in use.
+		return null;
+	}
+
+	@Override
+	public void place(Coordinates coordinates, int size) {
+		// Not in use.
```

Comment:
Shouldnt this method do something?

Reply:
It's only used in DiagramClass but since both it and Message implement Renderable, the method needs to be implemented.

Reply:
I think that it should be implemented if it has to be there in case someone has to use it

Reply:
again, part of the interface

Reply:
This is in the message class so it should be implemented